### PR TITLE
Upgrade guide

### PR DESCRIPTION
**Pull request by lemieux**
There it is.

@alavers @jpjoyal

### CODE REVIEW

**Comment by jugarrit** [comment]
image

**Comment by jugarrit** [comment]
http://i.imgur.com/RG0BS1U.gif

File: UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Upgrade guide to Smooch
+
+SupportKit is now Smooch!
+
+To reflect this transition, we bumped up the major version to 2.0. We introduced a lot of breaking changes in the internals of the library to let it hit our brand new public API.
```
Comment (alavers) [inline UPGRADE.md]: Announcing breaking changes sounds harsh and might discourage people from adopting it. In reality, none of the SDK methods have changed, so moving to smooch should be easy for most integrators. Avoiding the term breaking changes might communicate this better. EG:

```With Smooch, the internals of the SDK have changed. It is now using our public API (<--make this a hotlink) and as such we've bumped the major version to 2.0```

Comment (lemieux) [inline UPGRADE.md]: done.

File: UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+The most notable change is that you no longer use the `SupportKit` global variable, but the new `Smooch` global.
+
+```javascript
+var Smooch = require('smooch');
+
+Smooch.init({appToken: 'your_app_token'});
+```
+
+The rest is pretty much the same! We didn't change the CSS classes, so your customization should be safe.
```
Comment (jpjoyal) [inline UPGRADE.md]: I'm hesitant about not changing those. those will be the only sk remaining in the web sdk

Comment (lemieux) [inline UPGRADE.md]: What's the new prefix?

And yeah, it wouldn't break anything.. it would just be part of the upgrade process.

Comment (dannytranlx) [inline UPGRADE.md]: I don't see the issue of keeping `sk-`, we have a few over the other SDKs too!

Comment (spasiu) [inline UPGRADE.md]: :+1:

Comment (lemieux) [inline UPGRADE.md]: talked with @jpjoyal , no change needed

**Comment by dannytranlx** [comment]
:+1:

**Pull request merged**